docs/documentation/helm.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+Dekorate also supports generating manifests for `helm`. To make use of this feature you need to add:
+
+```xml
+<dependency>
```

Comment:
General remark for the Dekorate doc. If a gradle user is interested to use Dekorate, is there a section mentioning what they should do to configure also the gradle file managing the dependencies !!

Reply:
+100 
I created this issue to document Dekorate using Gradle: https://github.com/dekorateio/dekorate/issues/849